lib/gettext-setup/pot.rb

```diff
@@ -71,12 +71,11 @@ def self.generate_new_pot(locales_path = GettextSetup.locales_path, path = nil)
 
     def self.generate_new_po(language, locales_path = GettextSetup.locales_path,
                              pot_file = nil, po_file = nil)
+      language ||= ENV['LANGUAGE']
```

Comment:
Kind of a nit-pick, but can you put this under the `initialize` with the other variables we're trying to fill?